### PR TITLE
Implement edition numbering

### PR DESCRIPTION
**Pull request by tijmenb**
This PR adds an edition number to documents. This number will be shown in the document history, where it increases every time a new draft is created.

## What it looks like

screen shot 2018-09-19 at 15 14 42

https://trello.com/c/TVcThaKi

### CODE REVIEW

File: app/views/documents/show/_history_tab.html.erb [hunk 1]
```diff
@@ -1,5 +1,12 @@
+<% displayed_edition_number = nil %>
+
 <% @document.timeline_entries.includes(:user).order(created_at: :desc).each do |entry| %>
   <div class="timeline-entry govuk-!-margin-bottom-6">
+    <% if displayed_edition_number != entry.edition_number %>
+      <h3 class="govuk-heading-m"><%= entry.edition_number.ordinalize %> edition</h3>
```
Comment (kevindew) [inline app/views/documents/show/_history_tab.html.erb]: Should this go through the translations file?

File: app/controllers/documents_controller.rb [hunk 2]
```diff
@@ -23,6 +23,11 @@ def show
 
   def update
     document = Document.find_by_param(params[:id])
+
+    if document.publication_state == "sent_to_live"
+      document.current_edition_number = document.current_edition_number + 1
```
Comment (emmabeynon) [inline app/controllers/documents_controller.rb]: Can we do `document.current_edition_number += 1` here?

File: app/views/documents/show/_history_tab.html.erb (hunk 1 above)
Comment (tijmenb) [reply app/views/documents/show/_history_tab.html.erb]: 👍 done

File: app/controllers/documents_controller.rb (hunk 2 above)
Comment (tijmenb) [reply app/controllers/documents_controller.rb]: 👍 done

Comment (tijmenb) [reply app/controllers/documents_controller.rb]: Turns out `increment` exists as well! https://apidock.com/rails/ActiveRecord/Persistence/increment

**Pull request merged**